playbooks/byo/openshift-checks/health.yml

```diff
@@ -1,3 +1,10 @@
 ---
 - include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
```

Comment:
> using `tags: always` allows someone to run the playbook specifying a tag or tags to run a certain subsection of the playbooks, but will always run the plays/tasks with the `always` tag. Since these playbooks are part of initialization, we always want them to run.

There is a common property between the `openshift-check` playbooks involved here and others such as [playbooks/byo/openshift-cluster/config.yml](https://github.com/openshift/openshift-ansible/blob/b107f677a3563f6db9a1ef17fe10518d321d495c/playbooks/byo/openshift-cluster/config.yml#L1-L14): they all have 3 plays and all of them are includes. The first two plays/includes are related to initialization, while the third is the "main part" that includes a similarly named playbook under `common`.

We add `tags: always` to the first two includes, but NOT to the third one.

IIUC, the effect is that using the `--tags` flag to `openshift-ansible` causes the initialization to run (taking a non-trivial amount of seconds/minutes), and then possibly nothing else runs. In the case of `openshift-check`, certainly nothing else runs. I do not understand how that is a desirable behavior.